.gitignore

```diff
@@ -1 +1 @@
-node_modules
+/node_modules
```

Comment:
Yep it means that we only ignore the top level node_modules folder. Otherwise it will also ignore the test fixtures.